demo-site/servers/nodejs/px_config.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Lots of these files you don't need - for instance, there's no need to have a `shared_config.json` and a `px_config.json` and a `config.json` and a `config.inc.json`. Ultimately, all these are to generate one `px_config.json` file that's used for the sample site.

Reply:
removed most of them (only the unnecessary with safe remove